tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -52,7 +52,10 @@ sub run {
     $variables{HANA_CLIENT_SAR} = get_required_var("QESAPDEPLOY_IMDB_CLIENT");
     $variables{HANA_SAPCAR} = get_required_var("QESAPDEPLOY_IMDB_SERVER");
     $variables{ANSIBLE_REMOTE_PYTHON} = get_var("QESAPDEPLOY_ANSIBLE_REMOTE_PYTHON", "/usr/bin/python3");
-    $variables{HANA_DATA_DISK_TYPE} = get_var("QESAPDEPLOY_HANA_DATA_DISK_TYPE", "pd-ssd");
+    if (check_var('PUBLIC_CLOUD_PROVIDER', 'GCE')) {
+        $variables{HANA_DATA_DISK_TYPE} = get_var("QESAPDEPLOY_HANA_DISK_TYPE", "pd-ssd");
```

Comment:
Notice here I renamed the setting and I'm using a unique setting for both log and data disk type